packages/engine/Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -573,7 +576,16 @@ function handleZoom(
   const maxHeight = object.maximumZoomDistance;
 
   const minDistance = distanceMeasure - minHeight;
-  let zoomRate = zoomFactor * minDistance;
+
+  let fpsMultiplier = 1;
+  const fps = object.frameRateMonitor.averageFramesPerSecond;
+  // we set the ideal browser refresh rate to 30hz
+  if (fps) {
+    fpsMultiplier = 30 / fps;
```

Comment:
This is one open comment that I have not addressed: Including the target/ideal browser refresh rate as constant value (30 as in 30hz) in the code here. I am unsure if we should A) expose setting this in the API (but we already have zoomFactor in the public API) or B) set it in a more clear place, like a constants or system settings file.
I prefer option B

Reply:
I added a check to make sure `fps > 0`, which should ensure we avoid division by 0